lightningnetwork/lnd
# lnrpc: include a status indicator for channel verification operations

## Change Description
Description of change / link to associated issue.

## Steps to Test
1. Export a backup. 
2. Verify the backup with lncli verifychanbackup.
Response now includes a `success` indicator. 

## Pull Request Checklist
### Testing

### Code Style and Documentation

📝 Please see our  [Contribution Guidelines](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md) for further guidance.
